R/page.R

```diff
@@ -84,7 +84,8 @@ validateCssPadding <- function(padding = NULL) {
 #'   host URL of the page is displayed by default).
 #' @export
 page_navbar <- function(..., title = NULL, id = NULL, selected = NULL,
-                        sidebar = NULL, fillable = FALSE, fill_mobile = FALSE,
+                        sidebar = NULL, fillable = TRUE, fill_mobile = FALSE,
```

Comment:
Should `fill_mobile` be `fillable_mobile`?

Reply:
I did consider that, but I wasn't convinced it was significantly clearer, and it's more typing, so I opted for the shorter form

Reply:
Hmm my inclination is that if it means `fillable = TRUE` on mobile sizes, then I think consistency is important. And as an argument it's covered by autocomplete. But then I can also see that fillable on mobile could feel different from fillable on desktop and there's also the problem of changing the existing function name. So I guess in the end I'm less certain... but I do think consistency now will save us headaches in the long run